content/030_eksctl/test.md

```diff
@@ -17,7 +17,3 @@ STACK_NAME=$(eksctl get nodegroup --cluster eksworkshop-eksctl -o json | jq -r '
 ROLE_NAME=$(aws cloudformation describe-stack-resources --stack-name $STACK_NAME | jq -r '.StackResources[] | select(.ResourceType=="AWS::IAM::Role") | .PhysicalResourceId')
 echo "export ROLE_NAME=${ROLE_NAME}" | tee -a ~/.bash_profile
 ```
```

Comment:
Any reason why we are removing Congratulations comment? 

#### Congratulations!

You now have a fully working Amazon EKS Cluster that is ready to use!

Reply:
Hey @dalbhanj I am trying to limit troubleshoot related to attendees skipping the next step in the cluster deployment (the aws auth configuration).

I have run at least 3 EKS workshops where participants disregard the aws auth step because they see the "Congratulations" message and go straight to the next lab in the agenda. Then, when they are trying to see cluster info in the EKS console in later labs, we have to stop to troubleshoot what the issue is.

The aws auth instruction page ends with confirming to the attendee that their cluster is ready for the remainder of the labs. If you all prefer to keep the message, by all means restore it.

Reply:
@dalbhanj the Congratulations message is restored with a message reminding attendees to complete the Console Credentials step before moving on to other labs. Please review.